Sources/Cells/FavoriteAdTableViewCell/FavoriteAdTableViewCell.swift

```diff
@@ -0,0 +1,197 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+public protocol FavoriteAdTableViewCellDelegate: AnyObject {
+    func favoriteAdTableViewCellDidSelectMoreButton(_ cell: FavoriteAdTableViewCell)
+}
+
+public class FavoriteAdTableViewCell: UITableViewCell {
+
+    // MARK: - Public properties
+
+    public weak var delegate: FavoriteAdTableViewCellDelegate?
+
+    public weak var remoteImageViewDataSource: RemoteImageViewDataSource? {
+        didSet {
+            remoteImageView.dataSource = remoteImageViewDataSource
+        }
+    }
+
+    public var loadingColor: UIColor?
+
+    // MARK: - Private properties
+
+    private lazy var addressLabel = label(withFont: .detail, textColor: .stone, numberOfLines: 2)
+    private lazy var titleLabel = label(withFont: .caption, textColor: .licorice, numberOfLines: 2)
+    private lazy var descriptionPrimaryLabel = label(withFont: .bodyStrong, textColor: .licorice, numberOfLines: 0)
+    private lazy var descriptionSecondaryLabel = label(withFont: .detailStrong, textColor: .licorice, numberOfLines: 0)
+
+    private lazy var stackView: UIStackView = {
+        let stackView = UIStackView(withAutoLayout: true)
+        stackView.axis = .vertical
+        stackView.spacing = 0
+        stackView.alignment = .leading
+        return stackView
+    }()
+
+    private lazy var remoteImageView: RemoteImageView = {
+        let imageView = RemoteImageView(withAutoLayout: true)
+        imageView.contentMode = .scaleAspectFill
+        imageView.layer.cornerRadius = 16
+        imageView.layer.masksToBounds = true
+        return imageView
+    }()
+
+    private lazy var moreButton: UIButton = {
+        let button = UIButton(withAutoLayout: true)
+        let image = UIImage(named: .more).withRenderingMode(.alwaysTemplate)
+        button.setImage(image, for: .normal)
+        button.imageView?.contentMode = .scaleAspectFit
+        button.tintColor = .stone
+        button.addTarget(self, action: #selector(moreButtonTapped), for: .touchUpInside)
+        return button
+    }()
+
+    private lazy var statusRibbon: RibbonView = {
```

Comment:
How about `statusRibbon = RibbonView(withAutoLayout: true)`?

Reply:
I'll see if I can do that easily for `RibbonView`👍  I have to make some changes there for it to work